bootstrap.lic

```diff
@@ -38,19 +58,29 @@ if args.wipe_constants
   exit
 end
 
-scripts_to_run = args.flex
+scripts_to_run = args.flex || []
```

Comment:
@MahtraDR I think this should solve for the following scenario you encountered on install:
```
Finished install!
 CR:starting:["drinfomon"]
 [bootstrap: ]
 BS::
 BS:Running
```
The above happened because `args.flex` variable is nil and `nil.empty?` is nil and so line 63 (`until scripts_to_run.empty?`) enters the loop.

By ensure `scripts_to_run` is not nil but rather empty then that should satisfy line 63